README.md

```diff
@@ -76,6 +77,15 @@ yarn test
 yarn cypress
 ```
 
+## Explain how to add a user to the database
+
+In order to run this project, your database will need to have a user in it. To add one, run the follwing commands:
+
+```bash
+bin/rails console
+User.create!(name: "test", email: "test@example.com", organisation_slug:"test-org")
```

Comment:
`bin/rails db:seed` will take care of this I think

Reply:
nice thanks David, I'll update the readme with this